packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface IDataStoreAliasMessage {
+    readonly internalId: string;
+    readonly alias: string;
+}
+
+export interface IDataStoreAliasMapping {
+    readonly suppliedInternalId: string;
+    readonly alias: string;
+    readonly aliasedInternalId: string;
+}
+
+export const isDataStoreAliasMessage = (
```

Comment:
Not sure why you export 2 interfaces and this function from here, when they all seem to be used only in one file that imports them - why not move to that file?

Reply:
[This file had more code in it in the big PR](https://github.com/microsoft/FluidFramework/pull/7940/files#diff-16f470bb6f4d528acdee07dbb8d7d0bc1b2fbc6961ec05ec12d63c23d6ad6576). Will move them in this PR

Reply:
`IDataStoreAliasMapping` is used across 3 files so I'd rather leave all of them here if there's no strong feelings.